### PR TITLE
Make elevator care about what the others send

### PR DESCRIPTION
**Pull request by Kytzis**

### CODE REVIEW

File: The-ELEVATOR/orders/utils.go [hunk 1]
```diff
@@ -14,11 +14,44 @@ func contains(s []int, v int) bool {
 	return false
 }
 
+func combineLists(a []int, b []int) []int {
+	result := []int{}
+	for _, el := range a {
+		if contains(result, el) {
```
Comment (Benjamin-A) [inline The-ELEVATOR/orders/utils.go:20]: use better iterators. is el an elevator?

**Review by Benjamin-A — APPROVED** [review]
Code looks good, but some minor code quality improvements can be implemented. Runs well, even with high packet loss.

File: The-ELEVATOR/lamps/lamps.go [hunk 2]
```diff
@@ -4,12 +4,11 @@ import (
 	"Project/driver-go/elevio"
 	"Project/logger"
 	"Project/structs"
-	"fmt"
 )
 
 func ButtonLampUpdate(matrix structs.OrderMatrix, ID int) {
 
-	logger.Info("Lamps", fmt.Sprintf("%+v", matrix))
+	//logger.Info("Lamps", fmt.Sprintf("%+v", matrix))
```
[the remark this answers is not in the document]
Comment (Kytzis) [reply The-ELEVATOR/lamps/lamps.go:12]: yes, it was just changed before I redid the logger stuff

File: The-ELEVATOR/network/network.go [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
 	"Project/network-go/network/peers"
 	"Project/structs"
+	"encoding/json"
+	"fmt"
 	"time"
 )
 
 var ID int
-var txDelay = 10 * time.Millisecond
+var txDelay = 100 * time.Millisecond
```
[the remark this answers is not in the document]
Comment (Kytzis) [reply The-ELEVATOR/network/network.go:14]: Gets resolved by #18  if we add settings

**Review by Benjamin-A — APPROVED** [review]
Good Job!!

**Pull request merged**